src/edu/buffalo/cse/jive/finiteStateMachine/models/TransitionBuilder.java

```diff
@@ -56,21 +58,54 @@ private void addColorTransition(State state1, State state2, String color) {
 		this.transitions.append(s);
 		addNewLine();
 	}
+	
+	private void addNoColorTransition(State state1, State state2) {
+		String s = "\"" + state1.toString() + "\"" + " --> " + "\"" + state2.toString() + "\"";
+		this.transitions.append(s);
+		addNewLine();
+	}
+	

```

Comment:
Refactored the snippet for the neutral representation of states and put it in the method addNoColorTransition as it was called from multiple places.